docs/contents/runbook.md

```diff
@@ -6,24 +6,34 @@ unless there is a change.
 Prow jobs are in the [eks-distro-prow-jobs repo](https://github.com/aws/eks-distro-prow-jobs/tree/main/jobs/aws/eks-distro).  
 All other changes are to this package. 
 
-### Minor Release
-1. **Prow Build Job**: Create and merge Prow build job for new release (build-1-20-postsubmits.yaml)
-1. **Create EKS Distro PR**: Create various release components:
+### Create new Kubernetes minor version
+1. **Prow Build Job**: Create and merge Prow build job for new release
+   * copy previous job, ex: build-1-20-postsubmits.yaml, and change RELEASE_BRANCH
+1. **Prow Presubmit Job**: Create presubmit for new release
+   * copy previous job, ex: kubernetes-1-20-presubmits.yaml, and change RELEASE_BRANCH
```

Comment:
Should these be in two separate PRs?

Reply:
they dont have to be.  I was debating on how to structure this.  Do you see each of these bullets as a new PR?  technically down below they need to make presubmit jobs for all the projects that are versioned.  Those could all be in this same PR, but they could also be separate.  I open to suggestions.

Reply:
I interpret each bullet point as a PR, but idk if anyone else see it that way. It's fine if we keep them split and let people decide one or two PRs.

For the presubmit jobs on the projects, do you mean moving those changes up to here? I'd prefer those stay where they are because it'd be easier to remember which jobs you need to update when you're also updating the versions in this eks-distro

Reply:
Though I guess that means that step is two PRs (1 per repo), but again I think I might be alone in thinking bullet point == PR

Reply:
I sorta read them that way to tbh.  I think the other projects prow jobs make sense where they are in the instructions.  I am cool to leave it this way and we can see how people follow and if its too confusing, we can tweak.